README.md

```diff
@@ -16,7 +16,7 @@ features!
   support](#editor-support))
 - Vivify server starts lazily and automatically shuts down when no more viewers
   are connected
-- various [config options](#config)
+- various [customization options](docs/customization.md)
```

Comment:
Hope this doesn't get too drowned in the sea of links

Might be fine though

Reply:
I think it's fine; pretty sure someone who wants to customize something should be able to find it. If we end up with someone asking for something that's already there we can reconsider ^^